charts/pega/templates/_helpers.tpl

```diff
@@ -557,6 +557,14 @@ servicePort: use-annotation
   {{- end -}}
 {{- end -}}
 
+{{- define "hazelcastVersion" }}
+  {{- if and (not .Values.hazelcast.enabled)  .Values.hazelcast.clusteringServiceEnabled -}}
+    v5
+  {{- else -}}
+    v4
+  {{- end -}}
+{{- end -}}
```

Comment:
In the above 3 conditional blocks (hazelcastName, hzClusterName, hazelcastVersion), can these be simplified as below? LMK if its missing any case?
```
{- define "hazelcastVersion" }}
  {{- if .Values.hazelcast.enabled -}}
    v4
  {{- end -}}
{{- if  .Values.hazelcast.clusteringServiceEnabled -}}
    v5
  {{- end -}}
```